must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -156,3 +156,9 @@ echo "collecting dump of oc get volumereplication all namespaces" | tee -a  "${B
 echo "collecting dump of oc get volumereplicationgroups all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
 { oc get volumereplicationgroups --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/vrg_all_namespaces"
 { oc adm --dest-dir="${BASE_COLLECTION_PATH}/namespaces/all/" inspect vrg --all-namespaces --"${SINCE_TIME}"; } >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
+
+# Collect details of storageclassclaim of all namespaces for managed services
+echo "collecting dump of oc get storageclassclaim all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
+{ oc get storageclassclaim --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/storageclassclaim_all_ns"
```

Comment:
instead of `storageclassclaim_all_ns` use different files for each command?

Reply:
for that, we need to specify individual namespaces and collect them, as it was done in the previous commit, and for that, we need to get unique namespaces.  I would prefer this way as other resources are also collected in a similar way. Still, let me know if there is any specific reason to collect it in different files?

Reply:
i meant like below

```bash
echo "collecting dump of oc get storageclassclaim all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather
debug.log"
{ oc get storageclassclaim --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/get_storageclassclaim_all_ns"
{ oc describe storageclassclaim --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/desc_storageclassclaim_all_ns"
{ oc get storageclassclaim -oyaml --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/get_yaml_storageclassclaim_all_ns"
```

Reply:
oh, okay that. Yeah, that will be fine. Will update it.